modules/installation-disk-partitioning-upi-templates.adoc

```diff
@@ -35,6 +35,54 @@ Because `/var` must be in place before a fresh installation of {op-system-first}
 If you follow the steps to create a separate `/var` partition in this procedure, it is not necessary to create the Kubernetes manifest and Ignition config files again as described later in this section.
 ====
 
+.Prerequisites
+
+* If container storage is on the root partition:
```

Comment:
I'm thinking these prereqs can each be a single bullet? (And probably backticks rather than double quotes.) So for the first couple, something like:

> * If container storage is on the root partition, ensure that this root partition is mounted with `pquota` by including the `rootflags=pquota` option in the grub command line:
> * If the container storage is on a partition mounted by `systemd`, add the mount option as shown in the following example:

Reply:
+1